app/src/main/java/ch/heigvd/pro/b04/android/Home/EmojiGridAdapter.java

```diff
@@ -39,22 +47,26 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
         switch (position) {
             case 0:
                 break;
+            case 1:
+                break;
             default:
-                ((EmojiViewHolder) holder).bindEmoji(Emoji.values()[position - 1]);
+                ((EmojiViewHolder) holder).bindEmoji(Emoji.values()[position - 2]);
                 break;
         }
     }
 
     @Override
     public int getItemCount() {
-        return Emoji.values().length + 1;
+        return Emoji.values().length + 2;
     }
 
     @Override
     public int getItemViewType(int position) {
         return position == 0
                 ? VIEW_TYPE_HEADER
-                : VIEW_TYPE_EMOJI;
+                : (position == 1
+                    ? VIEW_TYPE_INSTRUCTION
+                    :VIEW_TYPE_EMOJI);
```

Comment:
Mostly a question of styling, but a `if ... else if ... else` might be easier to grasp.

Reply:
Changing this rn